vision/roadmap.md

```diff
@@ -101,7 +99,29 @@ Apps will be able to leverage an example server to subscribe users for push noti
 
 ## General Availability ✅
 
-Ready for production use. Check out [docs](https://xmtp.org/docs/client-sdk/javascript/concepts/intro-to-sdk), [tutorials](https://xmtp.org/docs/client-sdk/javascript/tutorials/build-an-xmtp-hello-world-app), and the [quickstart example](https://xmtp.org/docs/client-sdk/javascript/tutorials/quickstart).
+### Support for JavaScript and React apps
+
+Developers are able to build apps in JavaScript and React with the `xmtp-js` SDK:
+
+- [GitHub repo](https://github.com/xmtp/xmtp-js)
+- [Quickstart](/docs/client-sdk/javascript/tutorials/quickstart)
+- [Reference](/docs/client-sdk/javascript/reference/classes/Client)
+
+### Support for Swift mobile apps
+
+Developers are able to build mobile apps in Swift using the `xmtp-ios` SDK:
+
+- [GitHub repo](https://github.com/xmtp/xmtp-ios)
+- [Quickstart](/docs/client-sdk/swift/tutorials/quickstart)
+- [Reference](https://xmtp.github.io/xmtp-ios/documentation/xmtp/)
+
+### Support for Dart mobile apps
```

Comment:
This comment isn't related to the flutter/ios changes - @jhaaaa  what are your thoughts on reversing the order for the roadmap so GA items are at top and research at the bottom ?

This has been suggested by @prashantpbagga previously. I don't have a strong opinion